backend/Dockerfile

```diff
@@ -1,19 +1,12 @@
-FROM node:18-alpine
-
-# Set container working directory
+ARG NODE_IMAGE=lts-alpine
+FROM node:$NODE_IMAGE
 WORKDIR /app
+EXPOSE 3000
 
 RUN apk update
 RUN apk upgrade
 RUN apk add bash
 
-RUN npm install -g npm@8.8.0
-
-EXPOSE 3000
-
 COPY ./docker-entrypoint.sh /tmp/
 
-ARG WATCH=1
```

Comment:
Don't you use it for development?

Reply:
Yes, but it shouldn't be necessary in the build phase. As long as `BACKEND_DEBUG` is set, we can still choose to run the backend in watch mode or not

But if you're using it, then we can leave it in